src/components/item/header/MobileItemHeaderActions.tsx

```diff
@@ -0,0 +1,181 @@
+  const canEdit = isItemUpdateAllowedForUser({
+    memberships,
+    memberId: member?.id,
+  });
+  const canAdmin = member?.id
+    ? getHighestPermissionForMemberFromMemberships({
+        memberships,
+        memberId: member?.id,
+      })?.permission === PermissionLevel.Admin
+    : false;
+
+  const onClickChatbox = () => {
+    setIsChatboxMenuOpen(!isChatboxMenuOpen);
+  };
+
+  const toggleActionsDrawer = () => {
+    setIsItemActionsDrawerOpen(!isItemActionsDrawerOpen);
+  };
+  const closeDrawer = () => setIsItemActionsDrawerOpen(false);
+  const renderItemActions = () => {
+    // if id is defined, we are looking at an item
+    if (item && item?.id) {
+      // show edition only for allowed types
+      const showEditButton =
+        !editingItemId &&
+        ITEM_TYPES_WITH_CAPTIONS.includes(item.type) &&
+        canEdit;
+
+      const activeActions = (
+        <>
+          {showEditButton && (
+            <EditButton
+              item={item}
+              onClick={closeDrawer}
+              type={ActionButton.MENU_ITEM}
+            />
+          )}
+          <Box width="100%">
+            <MobileItemMenu
+              item={item}
+              canEdit={showEditButton}
+              closeDrawer={closeDrawer}
+              canAdmin={canAdmin}
+            />
+          </Box>
+
+          <Divider />
+        </>
+      );
+
+      return (
+        <>
+          {openedActionTabId !== ItemActionTabs.Settings && activeActions}
+
+          <DownloadButton
+            id={item.id}
+            name={item.name}
+            type={ActionButton.MENU_ITEM}
+            onClick={closeDrawer}
+          />
+          {canEdit && (
+            <ItemSettingsButton
+              id={item.id}
+              title={translateBuilder(BUILDER.SETTINGS_TITLE)}
+              type={ActionButton.MENU_ITEM}
+            />
+          )}
+        </>
+      );
+    }
+    return null;
+  };
+
+  return item ? (
+    <>
+      <Box
+        display="flex"
+        sx={{ '& button': { padding: 0.5 } }}
+        alignItems="center"
+      >
+        <ItemMetadataButton itemId={item.id} />
+        <ModeButton />
+        <ChatboxButton
+          tooltip={translateBuilder(BUILDER.ITEM_CHATBOX_TITLE)}
+          id={ITEM_CHATBOX_BUTTON_ID}
+          onClick={onClickChatbox}
+        />
+        <IconButton
+          onClick={toggleActionsDrawer}
+          id={MOBILE_MORE_ACTIONS_BUTTON_ID}
+        >
+          <MoreVertIcon />
+        </IconButton>
+      </Box>
+
+      <SwipeableDrawer
+        anchor="bottom"
+        open={isItemActionsDrawerOpen}
+        onClose={closeDrawer}
+        onOpen={() => setIsItemActionsDrawerOpen(true)}
+        PaperProps={{ sx: { maxHeight: '80vh', paddingY: 2 } }}
+      >
+        <Typography px={3} pb={1} variant="h5">
+          {item.name}
+        </Typography>
+        <Divider sx={{ borderColor: theme.palette.grey[400] }} />
```

Comment:
I do not think you need a divider here. if you really need one you can simply change the opacity:
```suggestion
        <Divider sx={{ opacity: 0.8 }} />
```
for example.